lxd/migration/migration_volumes.go

```diff
@@ -205,7 +205,10 @@ func MatchTypes(offer *MigrationHeader, fallbackType MigrationFSType, ourTypes [
 				}
 			}
 
-			if offer.Refresh != nil && *offer.Refresh {
+			// Only check for optimized refresh features if refreshing.
+			// It should also be considered that offer.Refresh can be nil for older LXD versions.
+			// In that case, optimized refresh won't be done using zfs or btrfs.
```

Comment:
The comment seems to not align with the code. If refresh in offer is nil, but refresh mode is enabled, why would we check for optimized refresh support (when the comment says we won't)

Reply:
I removed the last sentence.